website/client/src/components/inventory/equipment/attributesPopover.vue

```diff
@@ -5,29 +5,30 @@
         {{ `${$t('lockedItem')}` }}
       </h4>
       <div
-        v-if="isWrongClass"
-        class="popover-content-text"
+        v-if="item.currency === 'gems'"
```

Comment:
so I think the comment I left is happening because I tested with a gold purchasable item while here we're checking only for gems, if it's expected just ignore my comment - I noticed that in the linked issue the example is made with a gold purchasable item

Reply:
ahh, interesting... ok, I missed that. Sounds like I have some more work to do, thanks!